### PR TITLE
Update graphrbac.rst

### PR DESCRIPTION
**Pull request by 4c74356b41**
are you serious?

### CODE REVIEW

File: doc/graphrbac.rst [hunk 1]
```diff
@@ -10,7 +10,7 @@ See :doc:`Resource Management Authentication <quickstart_authentication>`
 for details on getting a ``Credentials`` instance.
 
 You will also need the tenant id of the AD you want to manage. Could be the AD UUID or domain name.
-`You can follow this documentation to get it <https://msdn.microsoft.com/fr-fr/library/azure/ad/graph/howto/azure-ad-graph-api-operations-overview#TenantIdentifier>`__.
+`You can follow this documentation to get it <https://msdn.microsoft.com/en-us/library/azure/ad/graph/howto/azure-ad-graph-api-operations-overview#TenantIdentifier>`__.
```
Comment (brjohnstmsft) [inline doc/graphrbac.rst]: Links to MSDN and azure.com shouldn't have any locale ID in the path. That way, users get redirected to the right localized page based on their system locale.

**Comment by 4c74356b41** [comment]
well, its not me, who put fr-fr link in there

**Comment by lmazuel** [comment]
Hi @4c74356b41 
Thanks for your feedback. I'm actually a french guy and usually I pay attention, I missed this one while writing it. Sorry for that.

**Comment by 4c74356b41** [comment]
People make mistakes, thats absolutely okay ;) I was just really shocked by that, because I've never previously seen something like this ;)
Thanks!

**Pull request merged**